charts/athens-proxy/templates/secret.yaml

```diff
@@ -0,0 +1,27 @@
+kind: Secret
+apiVersion: v1
+metadata:
+  name: {{ template "fullname" . }}-secret
+type: Opaque
+data:
```

Comment:
This is supposed to be `stringData` no? Otherwise it is expecting to the be base64 encoded?